chat_exporter/chat_exporter.py

```diff
@@ -55,23 +55,26 @@ async def export(ctx):
         await ctx.channel.send(embed=transcript_embed, file=transcript_file)
 
 
-async def generate_transcript(channel: discord.TextChannel, tz_info="US/Eastern"):
+async def generate_transcript(channel: discord.TextChannel, tz_info="US/Eastern", messages: list = None):
```

Comment:
I don't see the point in this `messages` variable. What are you trying to achieve? - This seems like a personal modification, rather than something which would benefit the module. 

The only 'benefit' I can see is a method to allow for limiting channel history (which is coming to the re-write), however this is the wrong way to go about that, if that is what you're intending(?)

Reply:
It's mainly so that people can specify messages to save. For example, when using channel.purge, it returns a list of messages. I think people would like to be able to save purged messages to a log for example and using the chat-exporter this would be easy.

Reply:
Hmm, I may need to think a little further on this. I'm not exactly a fan of this method to do what you're wanting. (assuming) It works, but doesn't seem clean.

It may be better under a `raw_transcript` function. Where it just tries to create a transcript from raw message input like this. I'll have to investigate a little more.

Reply:
Moved it to raw_transcript function